ProfilerCore/Brofiler.h

```diff
@@ -1,5 +1,13 @@
 #pragma once
 
+#include "Types.h"
+
+#ifdef LINUX64
+#include "Linux/PublicCommonLinux.h"
+#else
+#include "Windows/PublicCommonWindows.h"

```

Comment:
Just looked at the content. It seems to be more GCC/MSVC specific rather than OS specific